### PR TITLE
Product: Summary of key stats

### PR DESCRIPTION
**Pull request by lukorito**
Add mock state to chain

### CODE REVIEW

**Review by Kelz29 — APPROVED** [review]

File: lib/chain/sections/chain_info.dart [hunk 1]
```diff
@@ -11,12 +13,17 @@ class ChainInfo extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final chainInfo = ref.watch(chainProvider).value;
+    print(chainInfo);
```
Comment (berriose) [inline lib/chain/sections/chain_info.dart:17]: cleanup

File: lib/chain/sections/chain_info.dart [hunk 2]
```diff
@@ -11,12 +13,17 @@ class ChainInfo extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final chainInfo = ref.watch(chainProvider).value;
```
Comment (KalervoHyyppa) [inline lib/chain/sections/chain_info.dart]: Remove `.value` and just do `final chainInfo = ref.watch(chainProvider)` and then implement a `.when` in the UI below

File: lib/chain/sections/chain_info.dart [hunk 3]
```diff
@@ -11,12 +13,17 @@ class ChainInfo extends HookConsumerWidget {
...
         color: Colors.white,
         borderRadius: BorderRadius.circular(10.0),
-        border: Border.all(color: const Color(0xFFE7E8E8), style: BorderStyle.solid, width: 1.0),
+        border: Border.all(
+            color: const Color(0xFFE7E8E8),
+            style: BorderStyle.solid,
+            width: 1.0),
       ),
       height: 408,
       child: Column(
```
Comment (KalervoHyyppa) [inline lib/chain/sections/chain_info.dart:28]: Wrap this in a `chainInfo.when`. You may have to do some research, here's the documentation: https://pub.dev/documentation/riverpod/latest/riverpod/AsyncValue-class.html

**Review by KalervoHyyppa — CHANGES_REQUESTED** [review]
Most of our providers will be using `AsyncValue`, so it's best to implement `.when` when nescassary

File: lib/chain/sections/chain_info.dart (hunk 3 above)
Comment (lukorito) [reply lib/chain/sections/chain_info.dart]: @KalervoHyyppa I have implemented the feedback as requested.

**Review by KalervoHyyppa — APPROVED** [review]
LGTM

**Pull request merged**